src/en/developer-testing.md

```diff
@@ -201,3 +201,74 @@ following:
 ```
     self.deployment.log.debug("Some debug message here.")
 ```
+
+## Unit testing a layered charm
+
+Amulet is a mature tool for deploying and testing a charm in a test environment.
+
+For layered charms, it is often desirable to be able to run some tests before the charm has been built, however. For example, you may wish to run unit tests as you write your code. Waiting for the charm to build so that amulet can run tests on it would introduce unnecessary delays into the unit testing cycle. What follows are some best practices for writing unit tests for a layered charm.
+
+### Create a separate tox ini file
```

Comment:
👍  FINALLY! Tox is mentioned in the testing story of our docs. Big +1 to this already